app/assets/stylesheets/common/admin/badges.scss

```diff
@@ -122,28 +178,26 @@
   }
 }
 
-.badge-groupings {
-  list-style: none;
-  margin: 0;
-  padding: 10px 3px;
-
-  .badge-grouping-item {
-    border-bottom: 1px solid #dfdfdf;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-  }
-
-  .badge-grouping {
-    .badge-grouping-name-input {
-      margin: 0;
+// mobile specific styles - mostly commmon style overrides
+// TODO move to mobile sheet once mobile view has a seprate template.
+.mobile-view {
+  .admin-badges {
+    .content-list {
+      flex: 0 0 100%;
+      ul {
+        max-height: 40vh;
+        margin-right: 0;
+      }
     }
-  }
-
-  .actions {
-    float: right;
-    .btn {
-      padding: 3px 6px;
+    .badge-intro {
+      flex: 0 1 75%;
+    }
+    .current-badge {
+      margin: 20px 0;
+      width: 100%;
+    }
+    input[type="text"] {
```

Comment:
would prefer a class here

Reply:
I'm with you 100% on this but I'm afraid that adding classes to the inputs here is only a temporary fix to the problem. It might be better to do this systematically for all inputs where we can define the different widths based on a class like so (rough example)

```
.w25 {
  width: 25%;
}
.w50 {
  width: 25%;
}
.w75 {
  width: 25%;
}
.w100 {
  width: 100%;
}
```
<input class="w25">
<input class="w50">
<input class="w75">
<input class="w100">
```

or some refined version of that.

Reply:
feel free to open an internal topic to discuss this with kris